tsconfig.json

```diff
@@ -1,10 +1,7 @@
 {
   "compilerOptions": {
     "outDir": "./dist",
-    "target": "es2017",
-    "lib": [
```

Comment:
I don't believe it's correct to remove the `lib` line.
https://github.com/microsoft/TypeScript/wiki/Node-Target-Mapping#node-10

Reply:
Ah good shout